src/service/dynamo.ts

```diff
@@ -134,8 +134,25 @@ class Dynamo implements DynamoService {
   constructor(url: string, authProvider?: () => Promise<string>) {
     this.url = url;
     this.authProvider = authProvider;
+
+    // Workaround to get a jump start on the API key creation (which can take ~30 seconds)
+    this._fetch(`${this.url}/v1/graph/job/create`, { method: "GET" });
   }
 
+  async retryFetch(input: RequestInfo, init?: RequestInit | undefined): Promise<Response> {
+    let response = await fetch(input, init);
```

Comment:
I realize this is not right. Fetch  will probably throw here when it gets the 403